OpenRA.Game/Network/Order.cs

```diff
@@ -60,15 +61,15 @@ public CPos TargetLocation
 		{
 			get
 			{
-				return Target.SerializableCell.HasValue ? Target.SerializableCell.Value : CPos.Zero;
+				return Target.SerializableCell ?? CPos.Zero;
 			}
 		}
 
 		public Player Player { get { return Subject != null ? Subject.Owner : null; } }
 
 		Order(string orderString, Actor subject, Target target, string targetString, bool queued, CPos extraLocation, uint extraData)
 		{
-			OrderString = orderString;
+			OrderString = orderString ?? "";
```

Comment:
The `new Order()` constructor leaves OrderString as null. Any other constructor called with an explicit null would do the same. This crashes on serialization as a null value cannot be written. Figured it was easiest to simply coalesce this to an empty string, but another approach may be more correct.